scripts/postinstall.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Automatically copy over required settings for vscode
+if [ ! -f .vscode/settings.json ];
+	then cp .vscode/settings.json.required .vscode/settings.json ;
+fi
+
+
+# Ensure apps-rendering dependencies are installed
+cd apps-rendering && \
+	if [ \"$CI\" = true ];
+		then yarn --frozen-lockfile ;
+		else yarn ;
+	fi
```

Comment:
Is this still useful do we think? 
As I recall we added this because at one point in time, there was a single storybook that built both DCR & AR stories. People would often post about storybook not building because they'd forgotten to install AR dependencies.
I think these storybooks are now separate? If so we might not get a lot of value from this!